modules/rtticketedit.php

```diff
@@ -29,147 +29,142 @@
+
+                    if (ConfigHelper::checkConfig('phpui.helpdesk_customerinfo')) {
+                        if ($ticket['customerid']) {
+                            $params = array(
+                                'id' => $id,
+                                'customerid' => $ticket['customerid'],
+                                'customer' => $info,
+                                'emails' => $emails,
+                                'phones' => $phones,
+                            );
+                            $mail_customerinfo = $LMS->ReplaceNotificationCustomerSymbols(ConfigHelper::getConfig('phpui.helpdesk_customerinfo_mail_body'), $params);
+                            $sms_customerinfo = $LMS->ReplaceNotificationCustomerSymbols(ConfigHelper::getConfig('phpui.helpdesk_customerinfo_sms_body'), $params);
+                        } else {
+                            $mail_customerinfo = "\n\n-- \n" . trans('Customer:') . ' ' . $ticket['requestor'];
+                            $sms_customerinfo = "\n" . trans('Customer:') . ' ' . $ticket['requestor'];
+                        }
+                    }
+
+                    $message = end($ticket['messages']);
+                    $message['body'] = str_replace('<br>', "\n", $message['body']);
+
+                    $params = array(
+                        'id' => $id,
+                        'queue' => $queue['name'],
+                        'customerid' => $ticket['customerid'],
+                        'status' => $ticket['status'],
+                        'categories' => $ticket['categorynames'],
+                        'priority' => $RT_PRIORITIES[$ticket['priority']],
+                        'deadline' => $ticket['deadline'],
+                        'service' => $ticket['service'],
+                        'type' => $ticket['type'],
+                        'subject' => $ticket['subject'],
+                        'body' => $message['body'],
+                    );
+                    $headers['Subject'] = $LMS->ReplaceNotificationSymbols(ConfigHelper::getConfig('phpui.helpdesk_notification_mail_subject'), $params);
+                    $params['customerinfo'] = isset($mail_customerinfo) ? $mail_customerinfo : null;
+                    $body = $LMS->ReplaceNotificationSymbols(ConfigHelper::getConfig('phpui.helpdesk_notification_mail_body'), $params);
+                    $params['customerinfo'] = isset($sms_customerinfo) ? $sms_customerinfo : null;
+                    $sms_body = $LMS->ReplaceNotificationSymbols(ConfigHelper::getConfig('phpui.helpdesk_notification_sms_body'), $params);
+
+                    $LMS->NotifyUsers(array(
+                        'queue' => $ticket['queue'],
+                        'mail_headers' => $headers,
+                        'mail_body' => $body,
+                        'sms_body' => $sms_body,
+                    ));
+                }
+
+                $SESSION->redirect('?m=rtticketview&id=' . $id);
+        }
+    }
 }
```

Comment:
Jak dla mnie to wygląda na modyfikację, która nie zachowuje wszystkiego co było do tej pory tylko część rzeczy usuwa :(

Reply:
A poco te fragmenty kiedy ten kod jest pod switch?

Reply:
Zaraz to dokładnie przejrzę może faktycznie coś zjadłem.

Reply:
Dodałem commit z brakującym kodem.
W którymś z następnych PR przerobię to wszystkie SESSION REDIRECT na opcję ze zmienną goback tak żeby to z innych formularzy też działało prawidłowo.